signals/weather.py

```
@@ -0,0 +1,256 @@
+            return self._error_result(f"Dedup DB init failed: {exc}")
+
+        # 2. Fetch raw alerts from NWS
+        raw_alerts = fetch_alerts()
+
+        # 3. Enrich with revenue impact data
+        enriched = [enrich_alert(a) for a in raw_alerts]
+
+        # 4. Filter duplicates
+        try:
+            new_alerts = filter_new_alerts(enriched)
+        except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)